[Improve] Rename k8s to Kubernetes (#363)
apache/incubator-streampark-website: docs/flink-k8s/1-deployment.md

* **Flink Base Docker Image**： Base Flink Docker Image Tag can be obtained from  [DockerHub - offical/flink](https://hub.docker.com/_/flink) .And user can also use private image when Docker Register Account owns `pull` permission of it.

* **Rest-Service Exposed Type**：Description of candidate values for native Flink Kubernetes configuration [kubernetes.rest-service.exposed.type](https://ci.apache.org/projects/flink/flink-docs-stable/docs/deployment/config/#kubernetes) ：
  * `ClusterIP`：ip that StreamPark can access；
  * `LoadBalancer`：resource of LoadBalancer should be allocated in advance， Flink Namespace own permission of automatic binding，and StreamPark can access LoadBalancer`s gateway；
  * `NodePort`：StreamPark can access  all Kubernetes nodes；

> * **Kubernetes Pod Template**： It`s Flink custom configuration of pod-template.The container-name must be flink-main-container. If the Kubernetes pod needs a secret key to pull the docker image, please fill in the information about

Reviewer: ```suggestion
* **Kubernetes Pod Template**: This is Flink's custom configuration of pod-template. The `container-name` must be `flink-main-container`. If the Kubernetes pod needs a secret key to pull the Docker image, please fill in the information about
```

> the secret key in the pod template file.The example pod-template is as below：

Reviewer: ```suggestion
the secret key in the pod template file. Below is an example of pod-template:
```

      - name: regsecret
    ```

> * **Dynamic Option**：It`s dynamic parameter of Flink on Kubernetes（part of parameters can also be defined in pod-template）which should start with -D.Details are in[Flink on Kubernetes parameters](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes)

Reviewer: ```suggestion
* **Dynamic Option**: Dynamic options of Flink on Kubernetes (part of parameters can also be defined in pod-template). Options should start with `-D`. Details can be found at [Flink on Kubernetes parameters](https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/deployment/config/#kubernetes).
```

## other configuration

> StreamPark parameter related to Flink-Kubernetes in `applicaton.yml` are as below.And in most condition, it is no need to change it.

Reviewer: ```suggestion
StreamPark parameters related to Flink-Kubernetes in `applicaton.yml` are as below. And in most conditions, there is no need to change it.
```
